taskcluster/ci/beetmover/kind.yml

```diff
@@ -10,15 +10,16 @@ transforms:
     - taskgraph.transforms.task:transforms
 
 kind-dependencies:
+    - build
     - signing
     - repackage-signing
 
-jobs:
+tasks:
     macos:
         worker-type: beetmover
         worker:
             chain-of-trust: true
-            max-run-time: 3600
+            max-run-time: 1800
         run-on-tasks-for: [github-release, action:release-promotion]
```

Comment:
Going forward, should we maybe remove the gh-release trigger?  (not in this pr) - I feel the new promotion hooks are super cool, especially the granularity between "promotion" and "shipping". At the same time gh-release now is redundant to "release-promotion", right?

Also so sorry for asking @ahal to have them just to propose to remove them 🙈 
cc @bakulf

Reply:
We could remove it. Running `release-promotion` with `ship-client` on a `release` task group works. Still, for the sake of consistency it might be good to remove the `release` trigger.